examples/1-execute-query-mutation/package.json

```diff
@@ -16,7 +17,7 @@
     "reason-urql": "link:../../"
   },
   "devDependencies": {
-    "bs-platform": "^5.0.3",
+    "bs-platform": "5.0.4",
```

Comment:
I decided to pin everything to `bs-platform 5.0.4` b/c I was getting errors about the root project using a different version of `bs-platform` than the example project. So keeping everything pinned to 5.0.4 now, but will keep upgrading. It's just a devDep, so I don't think this will affect users at all.